### DISCUSSION TITLE
Upcoming changes to Standard Names

### DISCUSSION THREAD

**Discussion by mkavulich** [ESCOMP/CCPPStandardNames#88] [Announcements]
We have some exciting updates to announce for the Standard Names repository!

First, the scope of the project has expanded quite significantly since this repository was first introduced by the CCPP project. Many projects outside of CCPP are now making partial or full use of the Standard Names listed. Therefore it feels appropriate that the repository should be renamed to something more general for use in the Earth Science Modeling community. We have not made a final decision on the name, but the current leading proposal is "ESMStandardNames" (Earth System Modeling Standard Names). We welcome suggestions in the comments! And there is no need to worry about sudden changes to your workflow: GitHub handles repository renames smoothly, so an alias will still exist to the old repository name for the foreseeable future.

Second, the DTC has received some funding this year for code management related to the Standard Names repository. We are using this opportunity to re-visit the rules to make them more explicit and consistent, to help make the process easier when new names are proposed. In addition, we are attempting to resolve inconsistencies that currently exist within the set of names. We expect that this will be quite an extensive and invasive change to the current list, so we also plan to introduce version numbers in the Standard Names XML, to aid with groups who may not want to move to the updated set of names immediately (and also make potential future changes easier). This work is occurring in a dedicated branch (`release/v1`); again to reduce the immediate impact on current users.

Please reach out to me (kavulich@ucar.edu) if you have any questions or concerns!

**Comment by rljacob** [comment]
Can you say a little more about how these names are used and contrast it with the [CF Standard Names](https://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html) ?  I think the difference is that these are supposed to be used internally within models while CF is for NetCDF output files.  That should be clarified.

**Reply by rljacob** [reply]
Oh I see that this extends CF.  https://github.com/ESCOMP/ESMStandardNames/blob/main/StandardNamesRules.rst "Standard names should be identical to those from the latest version of the [Climate and Forecast (CF) metadata conventions](https://cfconventions.org/standard-names.html) unless an appropriate name does not exist in that standard, or the adoption of said names leads to inconsistencies in the naming convention".
